tests/hpc/product_migration.pm

```diff
@@ -8,7 +8,7 @@
 # without any warranty.
 
 # Summary: Checking ability to migrate from SLE 12 with HPC module to SLE 12 HPC Product
-# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+# Maintainer:  qa-sle@suse.de
```

Comment:
I am afraid this is too generic. But as it has "migration" in the name and as I assume there is no dedicated team for HPC right now I suggest to use the migration team. I am not aware of a team ML so take PO Wei Gao ?

Reply:
@coolgw  ok for you ?